### PR TITLE
updated dtendr.yaml

### PR DESCRIPTION
**Pull request by dtendr**
updated teamprop2 and vidreview1

### CODE REVIEW

**Comment by ryansb** [comment]
Please fix your merge conflicts by pulling the latest upstream repository.

File: scripts/people/2014/fall/dtendr.yaml [hunk 1]
```diff
@@ -12,3 +12,9 @@ hw:
   teamprop1: http://dtendr.blogspot.com/2014/09/team-proposal.html
   commarchreport: http://dtendr.blogspot.com/2014/09/commarch-report.html
   commarchpreso: https://docs.google.com/a/g.rit.edu/presentation/d/1ITde9mhXxQ5-e_xwCRvTBWqmZd-Du2RKeqO8r3iWcoQ/edit#slide=id.g3c4d4d6c3_30
+<<<<<<< HEAD
```
Comment (ryansb) [inline scripts/people/2014/fall/dtendr.yaml]: Delete this line

File: scripts/people/2014/fall/dtendr.yaml [hunk 2]
```diff
@@ -12,3 +12,9 @@ hw:
...
+  teamprop2: http://dtendr.blogspot.com/2014/10/team-proposal-2.html
+  vidreview1: http://dtendr.blogspot.com/2014/10/rip-remix-manifesto-video-review.html
+  meetup2: http://dtendr.blogspot.com/2014/10/rocpy-second.html
+=======
```
Comment (ryansb) [inline scripts/people/2014/fall/dtendr.yaml]: Delete this line and the line below

**Comment by ryansb** [comment]
:+1:

**Comment by decause** [comment]
@ryansb++

:deciduous_tree:

**Pull request merged**